app/services/messages_queries_services.py

```diff
@@ -0,0 +1,69 @@
+from http import HTTPStatus
```

Comment:
Try to name this file to be related with the message controller, which i assume you created this services for, since the other file is "message_controller", you could use something like "message_services" to keep the pattern.